web_pivot_computed_measure/static/src/xml/web_pivot_computed_measure.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2020 Tecnativa - Alexandre Díaz
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<templates>
+
+    <t t-name="web_pivot_computed_measure.ComputedMeasureOperations">
+        <option name="sum" value="m1+m2">
+            Sum (m1 + m2)
+        </option>
+        <option name="sub" value="m1-m2">
+            Sub (m1 - m2)
+        </option>
+        <option name="mult" value="m1*m2">
+            Mult (m1 * m2)
+        </option>
+        <option name="div" data-format="float" value="m1/m2">
+            Div (m1 / m2)
```

Comment:
I wonder what would happen if m2 is zero. Did you handle that?

Reply:
![infinity](https://user-images.githubusercontent.com/731270/77189994-bfb4d180-6ad8-11ea-866d-bde509a911df.png)

Reply:
Seems good enough.